docs/metrics.md

```diff
@@ -241,6 +241,7 @@ For [MediaPipe Graphs](./mediapipe.md) execution there are 4 generic metrics whi
 | counter      | ovms_responses | Useful to track number of packets generated by MediaPipe graph. Keep in mind that single request may trigger production of multiple (or zero) packets, therefore tracking number of responses is complementary to tracking accepted requests. For example tracking streaming partial responses of LLM text generation graphs. |
 | gauge      | ovms_current_graphs | Number of graphs currently in-process. For unary communication it is equal to number of currently processing requests (each request initializes separate MediaPipe graph). For streaming communication it is equal to number of active client connections. Each connection is able to reuse the graph and decide when to delete it when the connection is closed. |
 | counter      | ovms_graph_error | Counts errors in MediaPipe graph execution phase. For example V3 LLM text generation fails in LLMCalculator due to missing prompt - calculator returns an error and graph cancels. |
+| histogram      | ovms_graph_processing_time_us | Time for which mediapipe graph was opened and has been successfully closed. |
```

Comment:
I think in this table we should have also a column indicating for which type of request is applicable each metric. Right know everything is mixed. Some are for MP graphs only and some for individual model only. It could be adjusted in a separate PR.

Reply:
Every metric in this table should only be relevant for mediapipe